src/View/Helper/PhoneNumberFormat.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\I18n\PhoneNumber\View\Helper;
+
+use Laminas\I18n\PhoneNumber\Exception\ExceptionInterface;
+use Laminas\I18n\PhoneNumber\Exception\InvalidArgumentException;
+use Laminas\I18n\PhoneNumber\PhoneNumberValue;
+
+use function preg_match;
+use function strtoupper;
+
+final class PhoneNumberFormat
```

Comment:
@gsteel 
Maybe a topic for laminas-view 3: a view helper as a generic filter that allows all registered filters to be used with a chain. No need for extra formatter view helpers.

Reply:
Possibly? Would you define an 'AbstractFilterHelper' and then encourage a configuration based approach for individual use cases? Or, a bit like exposing a plugin manager in the view so `$this->filterHelper()->{$filterAlias}($value)` - The latter doesn't allow for chains without introducing loads of state and requires magic, but the former would be trivial to write from scratch for specific use cases and would offer better type safety.

In this case, the filters _and_ the view helper are just conveniences that proxy to methods in the phone number value object.

This conversation has me wondering whether a form element(s) should be introduced in this package too?

Reply:
I had the following in mind:

```php
echo $this->filter('  <i>example</i>  ')->stripTags()->trim()->stringToUpper();
```
…or something similar.

> This conversation has me wondering whether a form element(s) should be introduced in this package too?

Good idea! 👍

Reply:
https://github.com/laminas/laminas-view/pull/164